dali/test/python/test_operator_squeeze.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from nvidia.dali import pipeline_def
+import nvidia.dali.fn as fn
+import numpy as np
+from nose.tools import assert_raises
+
+
+def get_data(shapes):
+    return [np.empty(shape, dtype = np.uint8) for shape in shapes]
+
+@pipeline_def
+def squeeze_pipe(shapes, axes=None, axis_names=None, layout=None):
+    data = fn.external_source(lambda: get_data(shapes), layout=layout, batch=True, device = "cpu")
+    return fn.squeeze(data, axes=axes, axis_names=axis_names)
+
+def _testimpl_squeeze(axes, axis_names, layout, shapes, expected_out_shapes, expected_layout):
+    batch_size = len(shapes)
+    pipe = squeeze_pipe(batch_size=batch_size, num_threads=1, device_id=0, shapes=shapes, axes=axes, axis_names=axis_names, layout=layout)
+    pipe.build()
+    for _ in range(3):
+        outs = pipe.run()
+        assert outs[0].layout() == expected_layout
+        for i in range(batch_size):
+            out_arr = np.array(outs[0][i])
+            assert out_arr.shape == expected_out_shapes[i]
+            
+
+def test_squeeze():
+    # axes, axis_names, layout, shapes, expected_out_shapes, expected_layout
+    args = [
+        ([1], None, "XYZ", [[300, 1, 200], [10, 1, 10]], [(300, 200), (10, 10)], "XZ"),
```

Comment:
nitpick: You are using lists for shape and tuples for expected_shape. Can you use tuple (or list) in both cases?